src/AWSS3.jl

```diff
@@ -217,7 +234,36 @@ function s3_exists(
         end
     end
 end
+s3_exists_versioned(a...; kw...) = s3_exists_unversioned(global_aws_config(), a...; kw...)
+
+"""
+    s3_exists_unversioned([::AbstractAWSConfig], bucket, path; kwargs...)
+
+Returns a boolean whether an object exists at  `path` in `bucket`.
+
+See [`s3_exists_versioned`](@ref) to check for specific versions.
+"""
+function s3_exists_unversioned(aws::AbstractAWSConfig, bucket, path; kw...)
+    return (endswith(path, "/") ? _s3_exists_dir : _s3_exists_file)(
+        aws, bucket, path; kw...
+    )
+end
+s3_exists_unversioned(a...; kw...) = s3_exists_unversioned(global_aws_config(), a...; kw...)
 
+"""
+    s3_exists([::AbstractAWSConfig], bucket, path; version=nothing, kwargs...)
+
+Returns a boolean whether an object exists at  `path` in `bucket`.
+"""
+function s3_exists(
+    aws::AbstractAWSConfig, bucket, path; version::AbstractS3Version=nothing, kw...
+)
+    if version ≡ nothing
+        s3_exists_unversioned(aws, bucket, path; version=version, kw...)
```

Comment:
Maybe?
```suggestion
        s3_exists_unversioned(aws, bucket, path;, kw...)
```
Also what are the keywords we're passing on? I don't see any used anywhere so maybe we should just remove them